src/tutorial/furthertopics/message-triggers.rst

```diff
@@ -287,27 +284,24 @@ triggers another task bar and when fully completed triggers another task, baz.
 
          .. code-block:: cylc
 
-            [scheduler]
-            UTC mode = True
-
             [meta]
             title = "test workflow to demo message triggers"
 
+            [scheduler]
+            UTC mode = True
```

Comment:
```suggestion
                UTC mode = True
```

Reply:
I've gone through this doc and fixed a few other instances.